### PR TITLE
Enable login flow to work with access token and local logins

### PR DESCRIPTION
**Pull request by kushalmalani**
## Description

> Describe the purpose of this pull request.

Enable login flow to work with access token and local logins. 

**Note: We are pushing code to this branch for alpha rollout only. We would be maintaining different modes when we start with private beta release.**

## 🎟 Issue(s)

Resolves astronomer/astrohub-api#333

## 🧪 Functional Testing

> List the functional testing steps to confirm this feature or fix.

## 📸 Screenshots

> Add screenshots to illustrate the validity of these changes.

Local Login:

Screen Shot 2021-04-07 at 3 06 56 PM

Social Login:

Screen Shot 2021-04-07 at 3 09 06 PM

## 📋 Checklist

- [x] Rebased from the main (or release if patching) branch (before testing)
- [ ] Ran `make test` before taking out of draft
- [ ] Added/updated applicable tests
- [ ] Tested against [Houston-API](https://github.com/astronomer/houston-api/) and [Astronomer](https://github.com/astronomer/astronomer/) (if necessary).
- [ ] Communicated to/tagged owners of respective clients potentially impacted by these changes.
- [ ] Updated any related [documentation](https://github.com/astronomer/docs/)

### CODE REVIEW

**Comment by kushalmalani** [comment]
The test is failing, since its try to fetch a query from houston instead of astrohub-api. This entire branch is temporary and only for alpha roll out

**Comment by schnie** [comment]
@kushalmalani what do you think about creating a second client for gen2 operations? I think think the flows will diverge so having a separate client gives us a fresh slate to work with.

**Comment by kushalmalani** [comment]
It would make development a lot easier as we move forward as we dont have to worry about not disrupting gen1 operations while maintaining support for both gen1 and gen2 from the same repository.

How do we do releases? what happens when folks migrate from gen1 to gen2? This is seems more than just engineering decision to me. I am in for a new client.

File: astrohub/types.go [hunk 1]
```diff
@@ -0,0 +1,223 @@
+package astrohub
+
+import (
+	"fmt"
+
+	"github.com/Masterminds/semver"
+)
+
+// Response wraps all houston response structs used for json marashalling
```
Comment (schnie) [inline astrohub/types.go:9]: Astrohub

**Review by schnie — APPROVED** [review]

**Pull request merged**